lib/commands/component.js

```diff
@@ -73,16 +73,17 @@ exports.builder = (yargs) => {
     describe: `Change component file extension`,
     type: "string",
     default: "js",
-    choices: ["js", "jsx"],
+    choices: ["js", "jsx", "tsx"],
```

Comment:
@iamtabrezkhan and also I think here in the choices `choices: ["js", "jsx", "tsx"],`  one more option `ts` should be there so final line 76 should be like this `choices: ["js", "jsx","ts",  "tsx", ],`